packages/amazonq/test/e2e_new/amazonq/framework/cleanupHelper.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { By, WebviewView } from 'vscode-extension-tester'
+
+/* Finds all the tabs by looking for the close buttons and then closes them one by one.
+
+Logic:
+There is a button with the css mynah-tabs-close-button, we need to click that button and 
+close all the tabs after the test is done to avoid memory from a previous test. To double
+check if all the tabs are closed, we can check if the mynah-tabs-container is empty. */
+
+export async function closeAllTabs(webview: WebviewView) {
+    try {
+        const closeButtons = await webview.findWebElements(By.css('.mynah-tabs-close-button'))
+
+        for (const button of closeButtons) {
+            await button.click()
+            await new Promise((resolve) => setTimeout(resolve, 500))
+        }
+
+        // double check that all tabs are closed by checking if the mynah-tabs-container is empty
+        const tabsContainer = await webview.findWebElements(By.css('.mynah-tabs-container'))
+        if (
+            tabsContainer.length === 0 ||
+            (await tabsContainer[0].findElements(By.css('.mynah-tab-item-label'))).length === 0
+        ) {
+            console.log('All chat tabs successfully closed')
+        }
+    } catch (error) {
+        console.log('Error closing tabs:', error)
```

Comment:
if we can't close a tab, should we rethrow the error?

I'm thinking that if there's a bug where we can't close tabs, this test would still pass.